Features/links.feature

```diff
@@ -429,24 +429,21 @@ Functionaliteit: links bij levenscyclusendpoints
     Dan bevat _embedded de collectie van de voorkomens van de resource
     En bevat de _links.self van iedere resource voorkomen in de levenscyclus collectie, een link naar het voorkomen endpoint van het betreffende resource type, met de resource identificatie, versie en timestamp van registratie in de LV BAG
 
-  Scenario: link naar gerelateerde resources van een resource in een levenscyclus
-    Als de levenscyclus van een resource wordt opgevraagd via de levenscyclus endpoint van dat resource type
+  Scenario: link naar gerelateerde resources van een resource in een levenscyclus (niet zijnde een woonplaats)
+    Als de levenscyclus van een resource (niet zijnde een woonplaats) wordt opgevraagd via de levenscyclus endpoint van dat resource type
     Dan bevat _embedded de collectie van de voorkomens van de resource
     En bevat de _links van iedere resource voorkomen in de levenscyclus collectie, een attribuut met een link naar het levenscyclus endpoint van het betreffende gerelateerde resource type
 
-  Scenario: self link van een embedded resource in een resource 
-    Als de levenscyclus van een resource wordt opgevraagd via de levenscyclus endpoint van dat resource type
-    En één, meer of alle gerelateerde resources worden geëxpand met de expand parameter
+  Scenario: link naar gerelateerde resources van een resource in een levenscyclus (zijnde een woonplaats)
+    Als de levenscyclus van een woonplaats resource wordt opgevraagd via de levenscyclus endpoint van dat resource type
     Dan bevat _embedded de collectie van de voorkomens van de resource
-    En bevat ieder van de resources een _embedded sectie voor de gerelateerde resources
-    En bevat _embedded.<gerelateerder resource>._links sectie een self property met een link naar de levenscyclus endpoint van het gerelateerde resource type
+    En bevat de _links van iedere resource voorkomen in de levenscyclus collectie, een attribuut bronhouders met één of meer link(s) naar het voorkomen endpoint van het bronhouder type
 
-  Scenario: links naar gerelateerde resources in een embedded resources
+  Scenario: geen expand van gerelateerde resrouces van een resource in een levenscyclus
```

Comment:
Zou deze niet beter in de expand feature moeten worden opgenomen?

Reply:
Ja.